mne/channels/layout.py

```diff
@@ -631,8 +631,16 @@ def _auto_topomap_coords(info, picks):
         locs3d = np.array([eeg_ch_locs[ch['ch_name']] for ch in chs])
 
     # Duplicate points cause all kinds of trouble during visualization
-    if np.min(pdist(locs3d)) < 1e-10:
-        raise ValueError('Electrode positions must be unique.')
+    dist = pdist(locs3d)
+    if np.min(dist) < 1e-10:
+        problematic_electrodes = [
+            info['ch_names'][elec_i]
+            for elec_i in np.unique(squareform(dist < 1e-10).nonzero()[0])
```

Comment:
`squareform(pdist(locs) < 0.1).any(axis=0)` gives me a bool index... but that doesn't work in the plain list that is `info['ch_names']`.